### DISCUSSION TITLE
我在训练的时候遇到以下问题。

### DISCUSSION THREAD

**Discussion by boy-new-2019** [STVIR/pysot#351] [General]
训练集：coco
训练参数文件直接使用的experiments/siamrpn_r50_l234_dwxcorr_8gpu的config.yaml
报错如下：
![bug](https://user-images.githubusercontent.com/52913936/80170663-07200780-861b-11ea-9106-49790cc8b3d0.png)

**Comment by ZhiyuanChen** [comment]
It looks like a pytorch issue. 
Have you tried on different dataset?

**Comment by boy-new-2019** [comment]
> It looks like a pytorch issue.
> Have you tried on different dataset?

额，因为vpn流量有限，没有试过用其他的数据集跑，但是在之前使用test.py和demo时没报错，现在准备将pytorch从您推荐的0.4升至1.0试试

**Comment by ZhiyuanChen** [comment]
> > It looks like a pytorch issue.
> > Have you tried on different dataset?
> 
> 额，因为vpn流量有限，没有试过用其他的数据集跑，但是在之前使用test.py和demo时没报错，现在准备将pytorch从您推荐的0.4升至1.0试试

我映像中这个代码是可以支持到1.2的，1.3以上需要改几行loss，不过也不难

**Comment by ZhiyuanChen** [comment]
由于本议题没有得到更进一步的提问，我现在将其关闭。
请创建一个新的议题如果你有其他问题。
